types/params.go

```diff
@@ -106,5 +140,8 @@ func (params ConsensusParams) Update(params2 *abci.ConsensusParams) ConsensusPar
 	if params2.EvidenceParams != nil {
 		res.EvidenceParams.MaxAge = params2.EvidenceParams.MaxAge
 	}
+	if params2.ValidatorPubkeyTypes != nil {
```

Comment:
Works for structs but not sure if we might just end up with a non-nil empty list here?